src/psyclone/psyGen.py

```diff
@@ -6060,8 +6196,7 @@ def __init__(self, kernel=None):
         # Reference to KernelSchedule to which this symbol table belongs.
         self._kernel = kernel
 
-    def declare(self, name, datatype, shape=[], scope='local',
-                is_input=False, is_output=False):
+    def declare(self, name, datatype, shape=[], interface=None):
```

Comment:
Not your fault but pylint says: "[dangerous-default-value] Dangerous default value [] as argument"
We can use `shape=None` and then give the empty list if it is None.

Reply:
Oh yes - I'd already fixed that in the `Symbol` constructor so fixing it here is trivial :-)